src/posts/depricating-may-theorem.mdx

```diff
@@ -179,7 +182,7 @@ It's easy to say one thing in English while writing something entirely different
 <a name="repeating-mays-mistake-for-great-profit"></a>
 ## Repeating May's Mistake For Great Profit
 
-<center>***"Turn any boring definition into an exciting theorem with this one weird trick!"***</center>
+<center class='has-text-bold'>"Turn any boring definition into an exciting theorem with this one weird trick!"</center>
```

Comment:
Note the 3 asterisks that indicate this should be bold italic like in the original post:

![Screenshot 2024-04-09 at 8 19 03 AM](https://github.com/okTurtles/groupincome.org/assets/138706/fbfa8e0e-3901-4a9f-8b75-ac014bdacf50)

I'm wondering... is there a problem with how our markdown renderer is setup?

There's this issue, and the one above on line 163... A good markdown renderer should have rendered `\*` properly...

Ideally, our markkdown renderer would properly render both of these lines. Should I open a separate issue for that?

Reply:
@taoeffect I just turned this back to 
```
<center>***"Turn any boring definition into an exciting theorem with this one weird trick!"***</center>
```
It works properly and turns out I didn't need to change it in `.mdx` file.

It appears we do need to update `/* /* /*` to `* * *` in `.mdx` though. there seems to be no need to escape it there.

---

It's unclear what causes the original issue in the first place, but It apparently is related to enabling `@astrojs/mdx` in the config file. The project uses version `0.x.x` of `@astrojs/mdx` package while it's latest version is `2.x.x` so tried to update it to the latest but that is not simple and involves some breaking changes (requires astro `v2.x.x` -> `v4.x.x` too) like below.

<img src='https://github.com/okTurtles/groupincome.org/assets/17641213/b867b72b-0ffb-4e99-9834-6d79bd721437' width='80%'>

I can work on bumping up all these packages to the latest as a follow up work of this PR and see if using `v2.x.x` of `@astrojs/mdx` actually fixes the original issue in `.md` file too.